docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md

```diff
@@ -0,0 +1,137 @@
+- Pre-trained model. 
+    - Build Your Own [sentiment analysis machine learning model]()
+    - Download this [pre-trained sentiment analysis machine learning model]()
+
+## Create Azure Functions Project
+
+1. Open Visual Studio 2017. Select **File** > **New** > **Project** from the menu bar. In the **New Project** dialog, select the **Visual C#** node followed by the **Cloud** node. Then select the **Azure Functions** project template. In the **Name** text box, type "SentimentAnalysisFunctionsApp" and then select the **OK** button.
+1. In the **New Project** dialog, open the dropdown above the project options and select **Azure Functions v2 (.NET Core)**. Then, select the **Http trigger** project and then select the **OK** button.
+1. Create a directory named *MLModels* in your project to save your model:
+
+    In **Solution Explorer**, right-click on your project and select **Add** > **New Folder**. Type "MLModels" and hit Enter.
+
+1. Install the **Microsoft.ML NuGet Package**:
+
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML**, select that package in the list, and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed.
+
+## Add Pre-built Model To Project
+
+1. Copy your pre-built model to the *MLModels* folder.
+1. In Solution Explorer, right-click your pre-built model file and select **Properties**. Under **Advanced**, change the value of **Copy to Output Directory** to **Copy if newer**.
+
+## Create Function to Analyze Sentiment
+
+Create a class to predict sentiment. Add a new class to your project:
+
+1. In **Solution Explorer**, right-click the project, and then select **Add** > **New Item**.
+
+1. In the **Add New Item** dialog box, select **Azure Function** and change the **Name** field to *AnalyzeSentiment.cs*. Then, select the **Add** button.
+
+1. In the **New Azure Function** dialog box, select **Http Trigger**. Then, select the **OK** button.
+
+    The *AnalyzeSentiment.cs* file opens in the code editor. Add the following `using` statement to the top of *GitHubIssueData.cs*:
+
+```csharp
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Microsoft.ML;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+```
+
+### Create Data Models
+
+Create classes to define machine learning model's input and outputs. Add the following code to the *AnalyzeSentiment.cs* file under the *AnalyzeSentiment* class definition. 
```

Comment:
Create classes to define **the** machine learning model's input and outputs.

Reply:
Resolved